internal/installer/elasticagent_tar_macos.go

```diff
@@ -54,8 +56,13 @@ func (i *elasticAgentTARDarwinPackage) AddFiles(ctx context.Context, files []str
 // Inspect returns info on package
 func (i *elasticAgentTARDarwinPackage) Inspect() (deploy.ServiceOperatorManifest, error) {
 	return deploy.ServiceOperatorManifest{
+<<<<<<< HEAD
 		WorkDir:    "/opt/Elastic/Agent",
 		CommitFile: "/elastic-agent/.elastic-agent.active.commit",
+=======
+		WorkDir:    i.metadata.AgentPath,
+		CommitFile: "elastic-agent/.elastic-agent.active.commit",
+>>>>>>> 892e951c (Use Orka ephemeral workers to run the fleet testing (#2626))
```

Comment:
`AgentPath` is not backported to 8.2
```suggestion
		WorkDir:    "/opt/Elastic/Agent",
		CommitFile: "elastic-agent/.elastic-agent.active.commit",
```